### PR TITLE
Adds test coverage for a few missing methods

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: Post.all_tags [hunk 1]
```diff
@@ -0,0 +1,26 @@
+=> [[32m#<Post:0x0000000acf8bb8[0m
```
Comment (zamith) [inline Post.all_tags]: Should this be removed?

File: spec/controllers/application_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+RSpec.describe ApplicationController, type: :controller do
+  context '#admin_controller' do
+    it 'is false' do
+      expect(controller.admin_controller?).not_to be
+    end
+  end
+
+  context '#load_tags' do
+    before(:all) do
```
Comment (zamith) [inline spec/controllers/application_controller_spec.rb]: Why not `before :each`?

File: spec/helpers/nav_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe NavHelper, type: :helper do
```
Comment (zamith) [inline spec/helpers/nav_helper_spec.rb]: Could this tests be too brittle? Do we really need to test the classes that are generated?

File: Post.all_tags (hunk 1 above)
Comment (naps62) [inline Post.all_tags]: whaaat? :o

File: spec/controllers/application_controller_spec.rb (hunk 2 above)
Comment (naps62) [inline spec/controllers/application_controller_spec.rb]: because I got it wrong :smile:

File: spec/helpers/nav_helper_spec.rb (hunk 3 above)
Comment (naps62) [inline spec/helpers/nav_helper_spec.rb]: You're right. The original point of this was to test whether the inner block was called (which is equivalent to check for the existance of a link inside a list-item). I can just look for `li > a`

**Pull request merged**